Makefile

```diff
@@ -156,6 +156,7 @@ gen-manifests: manifest-templator
 	LINUX_BRIDGE_CNI_IMAGE=$(LINUX_BRIDGE_CNI_IMAGE) \
 	KUBEMACPOOL_IMAGE=$(KUBEMACPOOL_IMAGE) \
 	MACVTAP_CNI_IMAGE=$(MACVTAP_CNI_IMAGE) \
+	IMAGE_PULL_POLICY=IfNotPresent \
```

Comment:
why not change the default of generate-manifests.sh ?
https://github.com/kubevirt/cluster-network-addons-operator/blob/45dfbafbdf9b31f6753153a5fbaa8252d2d5d31d/hack/generate-manifests.sh#L8